djoser/constants.py

```diff
@@ -12,3 +12,4 @@ class Messages(object):
     INVALID_PASSWORD_ERROR = _("Invalid password.")
     EMAIL_NOT_FOUND = _("User with given email does not exist.")
     CANNOT_CREATE_USER_ERROR = _("Unable to create account.")
+    IN_ACTIVE_ACCOUNT_ERROR = _("Email not verified")
```

Comment:
Please change `IN_ACTIVE_ACCOUNT_ERROR` -> `INACTIVE_ACCOUNT_ERROR`here and further down the line